### PR TITLE
[JA] Translate Ecto/Querying

### PR DESCRIPTION
**Pull request by Tsuyoshi84**
I translated Ecto/Quering page.
Please review! @elixirschool/japanese

### CODE REVIEW

File: ja/lessons/ecto/querying.md [hunk 1]
```diff
@@ -0,0 +1,443 @@
+---
+version: 1.0.0
+title: クエリ
+---
+
+{% include toc.html %}
+
+このレッスンでは、 `Example` アプリと [前のレッスン](./associations) で設定した映画カタログドメインを作成します。
```
Comment (Tsuyoshi84) [inline ja/lessons/ecto/querying.md]: I translated the sentence `movie-cataloguing domain` as `映画カタログドメイン`, but there should be a better translation I think. Let me know if you have any idea!

**Comment by riseshia** [comment]
I wil check until this weekend :bow:

File: ja/lessons/ecto/querying.md [hunk 2]
```diff
@@ -0,0 +1,443 @@
+---
+version: 1.0.0
```
Comment (riseshia) [inline ja/lessons/ecto/querying.md]: We may say this v1.0.1 https://github.com/elixirschool/elixirschool/pull/1734/files#diff-42858e1c36d5073e0f1f6c9b1b86d979 (It seems already fixed / no need to fix in Japanese..?

File: ja/lessons/ecto/querying.md [hunk 3]
```diff
@@ -0,0 +1,443 @@
...
+
+```elixir
+iex> Movie |> last() |> Repo.one()
+```
+
+## 関連したデータのクエリ
+
+### プリロード
+
+`belongs_to` 、 `has_many` 、そして `has_one` マクロが公開する関連レコードにアクセスできるようにするために、関連したスキーマを _プリロード_ する必要があります。
```
Comment (riseshia) [inline ja/lessons/ecto/querying.md]: (maybe) マクロが公開する関連レコード -> マクロからの公開する関連レコード

File: ja/lessons/ecto/querying.md (hunk 1 above)
Comment (riseshia) [reply ja/lessons/ecto/querying.md]: I have no better idea too. 🙈 
or 映画ドメイン may be enough in this case..

(maybe) 設定した映画カタログドメインを作成します -> 設定した映画カタログドメインを利用します

File: ja/lessons/ecto/querying.md (hunk 2 above)
Comment (Tsuyoshi84) [reply ja/lessons/ecto/querying.md]: Oops, I didn't realize that the version has increased. It looks like the change has only removed unnecessary spaces, so there is nothing we have to do but just bump up the version I guess.

File: ja/lessons/ecto/querying.md (hunk 1 above)
Comment (Tsuyoshi84) [reply ja/lessons/ecto/querying.md]: OK, then I'll just translate it as `映画ドメイン`!
`映画カタログドメイン` sounds kind of strange and confusing so it should be avoided anyway 🤔

**Comment by riseshia** [comment]
🎉

**Pull request merged**